airflow/providers/google/cloud/operators/cloud_storage_transfer_service.py

```diff
@@ -1026,6 +1048,16 @@ def __init__(
         self.delete_job_after_completion = delete_job_after_completion
         self._validate_inputs()
 
+        if self.destination_path and not self.destination_path.endswith('/'):
```

Comment:
This check seems to be the same as the gcs_path(if Im not wrong), does it make sense to move it to a function?

Reply:
Actually I missed the function `_validate_inputs` under each operator class somehow, it's the right place for this check, moved it there